tests/foreman/api/test_syncplan.py

```diff
@@ -1088,7 +1088,7 @@ def test_positive_synchronize_custom_product_weekly_recurrence(self):
         """
         delay = 4 * 60
         start_date = datetime.utcnow() - timedelta(weeks=1) \
-            + timedelta(seconds=delay)
+            + timedelta(seconds=3600)
```

Comment:
You can use `delay * 60` to make it a little less of a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants).

Reply:
OK, fixing.